### PR TITLE
Change key vault to track2

### PR DESCRIPTION
**Pull request by Luyunmt**
Change key vault to track2.
@jongio for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@jongio - Can you help ping the corresponding person to review this PR?

**Comment by Tianyun-G** [comment]
@scottaddie, @AlexGhiondea, @antkmsft - Could you help review this PR?

**Comment by v-xuto** [comment]
@heaths - Could you please review this PR?

File: Common/Utilities.cs [hunk 1]
```diff
@@ -2980,38 +2979,6 @@ public static string GetCertificatePath(string certificateName)
             return Path.Combine(Utilities.ProjectPath, "Asset", certificateName);
         }
 
-        public static void SendMessageToTopic(string connectionString, string topicName, string message)
```
Comment (jongio) [inline Common/Utilities.cs:2983]: Is this just unused code?

**Review by heaths — APPROVED** [review]

File: Common/Utilities.cs (hunk 1 above)
Comment (Tianyun-G) [reply Common/Utilities.cs:2983]: @jongio - Yes, this code is unused, so I deleted.

**Pull request merged**